Lambdas/GHHooks/EventHandler/Handlers/ReleaseReporter.swift

```diff
@@ -72,12 +72,18 @@ struct ReleaseReporter {
     }
 
     func getPRsRelatedToRelease() async throws -> (Int, [SimplePullRequest]) {
-        let commits: [Commit]
-        if let tagBefore = try await self.getTagBefore() {
-            commits = try await self.getCommitsInRelease(tagBefore: tagBefore)
-        } else {
-            commits = try await self.getAllCommits()
-        }
+        let commits: [Commit] =
+            if let comparisonRange = self.findComparisonRange() {
+                try await self.getCommitsInRelease(
+                    comparisonRange: comparisonRange
+                )
```

Comment:
get the comparison range from `**Full Changelog**: https://github.com/DiscordBM/DiscordBM/compare/v1.12.2...v1.13.0` comments when possible.